apps/src/javalab/JavalabView.jsx

```diff
@@ -120,20 +133,26 @@ class JavalabView extends React.Component {
     return (
       <StudioAppWrapper>
         <div style={style.javalab}>
-          <div style={style.instructionsAndPreview}>
-            <InstructionsWithWorkspace>
-              <div style={style.preview}>
-                <PaneHeader hasFocus={true}>
-                  <PaneSection>Preview</PaneSection>
-                </PaneHeader>
-              </div>
-            </InstructionsWithWorkspace>
+          <div
+            id="visualizationColumn"
+            className="with_padding responsive pin_bottom"
```

Comment:
Most classnames I've come across are kebab case. Do we have a convention for casing of class names?

Reply:
Looks like those two classes aren't needed anyway. Removed!